### PR TITLE
docs: Add readme for popint subject

### PR DESCRIPTION
**Pull request by eslopfer**
Add readme for popint subject.

### CODE REVIEW

File: subjects/popint/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+```go
+package main
+
+import (
+    "piscine"
+    "fmt"
+)
+
+func main() {
+    ints := {6,7,8,9,}
```
Comment (Hamzaelkhatri) [inline subjects/popint/README.md:28]: add `ints := []int{6, 7, 8, 9}`

**Review by Hamzaelkhatri — CHANGES_REQUESTED** [review]

File: subjects/popint/README.md (hunk 1 above)
Comment (eslopfer) [reply subjects/popint/README.md:28]: Thanks

**Review by Hamzaelkhatri — APPROVED** [review]

File: subjects/popint/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+## popint
+
+### Instructions
+
+Write a function that receives a slice of int and returns a new slice without the last element.
```
Comment (MSilva95) [inline subjects/popint/README.md:5]: just missing explaining what happens when the slice is empty

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: subjects/popint/README.md (hunk 2 above)
Comment (eslopfer) [reply subjects/popint/README.md:5]: I'll add that. Thank you.

**Pull request merged**